### PR TITLE
Upgrade patches from Chromium 75.0.3770.38 to Chromium 75.0.3770.52

### PR DESCRIPTION
**Pull request by bsclifton**
Fixes https://github.com/brave/brave-browser/issues/4547
Related https://github.com/brave/brave-browser/pull/4544

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Windows
  - [ ] macOS
  - [ ] Linux
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on
  - [ ] Windows
  - [ ] macOS
  - [ ] Linux
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: patches/chrome-common-mac-app_mode_chrome_locator.mm.patch [hunk 1]
```diff
@@ -1,13 +1,39 @@
...
-   // However, we want the shims to be agnostic to distribution and operate based
-   // on the data in their plist, so encode the framework names here.
-   NSDictionary* framework_for_exe = @{
--    @"Chromium": @"Chromium",
+@@ -113,7 +113,34 @@ bool GetChromeBundleInfo(const base::FilePath& chrome_bundle,
+     }
+   }
+ 
++<<<<<<< ours
+   if (!framework_path_and_structure)
```
Comment (yrliou) [inline patches/chrome-common-mac-app_mode_chrome_locator.mm.patch]: Ongoing merge work here?

Comment (bsclifton) [reply patches/chrome-common-mac-app_mode_chrome_locator.mm.patch]: oh wow- this is interesting... Have never seen this before

Comment (bsclifton) [reply patches/chrome-common-mac-app_mode_chrome_locator.mm.patch]: Thanks for proactively grabbing the review on this @yrliou 😄 Basically, the Jenkins job that @mkarolin and I had put together freaked out on the merge here 😂 Looks like this patch is no longer needed! Just removed in a commit and will let CI run

**Comment by bsclifton** [comment]
Failed with:
```
17:11:49  ../../chrome/browser/download/download_permission_request.cc:39:36: error: use of undeclared identifier 'IDS_MULTI_DOWNLOAD_WARNING_TITLE'
17:11:49    return l10n_util::GetStringUTF16(IDS_MULTI_DOWNLOAD_WARNING_TITLE);
```

Will fix strings and update PR...

**Comment by mkarolin** [comment]
Closing in favor of #2545

**Pull request closed without merging**